libs/filabridge/include/private/filament/Variant.h

```diff
@@ -40,24 +40,33 @@ namespace filament {
         // SRE: Shadow Receiver
         // SKN: Skinning
         // DEP: Depth only
-        // FOG: Fog
+        // FOG: Fog (or picking)
```

Comment:
I'm confused what fog has to do with picking. If picking is enabled, doesn't that mean fog would have to be as well (and vice-versa)?

Reply:
So in fact it's because I use the same bit for both FOG and PICKING.  This is because you can never have the FOG bit set when the DEPTH bit is set. So, in fact, the DEPTH bit defines what the other bits mean.